advanced-javascript/tests/objects.test.js

```diff
@@ -1,9 +1,83 @@
-const assert = require('chai').expect;
-const objectFunctions = require('../src/objects');
+/* eslint-disable */
 
+const expect = require('chai').expect;
+const assert = require('chai').assert;
+const objectFunctions = require('../src/objects');
+const funcs = objectFunctions;
 // whoops.. there is no test suite for this file. You'll simply just have to create one :/
 describe('objects', () => {
-  it('should be an object', () => {
+  describe('keys', () => {
+  	it('should be a function', () => {
+  		assert.typeOf(funcs.keys, 'function');
```

Comment:
Everything looks awesome. I like the way your group mixed up using expect & assert throughout your tests.